events/bounty-hunters.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bounty Hunters
+---
+
+Bounty Hunters este un nou eveniment permanent activ pe server, vizibil în `/event` similar cu World Exploration. In fiecare zi, serverul va spawna pentru fiecare jucator la un interval de 8 ore (00:00, 08:00, 16:00) un numar de NPC-uri ce trebuiesc ucise.
+In functie de tipul contului detinut se vor spawna 8 NPC-uri (cont standard & Gold) sau 10 NPC-uri (cont Platinum).
+
+Exista 3 tip-uri de NPC-uri:
+"Peaceful" - nu risposteaza cand este atacat
+"Aggressive" - riposteaza cand este atacat
+"Aggressive with suitcase" - riposteaza cand e atacat si dropeaza o servieta la deces, servieta ce trebuie livrata la Alfred intr-un interval de 10 minute
+
```

Comment:
**Ce este "Bounty Hunters?"**

"Bounty Hunters" este un eveniment permanent activ pe server, vizibil în [/event] similar cu World Exploration. In fiecare zi, serverul va spawna pentru fiecare jucator la un interval de 8 ore (00:00, 08:00, 16:00) un numar de NPC-uri ce trebuiesc ucise. In functie de tipul contului detinut se vor spawna 8 NPC-uri (cont standard & Gold) sau 10 NPC-uri (cont Platinum). In cadrul acestui eveniment exista 3 tipuri de NPC-uri: „Peaceful” – nu riposteaza cand este atacat; „Aggressive” – riposteaza cand este atacat & „Aggressive with suitcase” – riposteaza cand este atacat si dropeaza o servieta ce trebuie livrata la Alfred intr-un interval de 10 minute.